library/src/main/scala-old-collections/scala/scalajs/js/WrappedMap.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.scalajs.js
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+import scala.collection.mutable.Builder
+import scala.scalajs.js
+
+/** Wrapper to use a js.Map as a scala.mutable.Map */
+@inline
+final class WrappedMap[K, V](private val underlying: js.Map[K, V])
+    extends mutable.AbstractMap[K, V]
+       with mutable.Map[K, V]
+       with mutable.MapLike[K, V, js.WrappedMap[K, V]] {
```

Comment:
```suggestion
    with mutable.Map[K, V]
    with mutable.MapLike[K, V, js.WrappedMap[K, V]] {
```
Sorry. I must have missed those in my previous review. I think I had not given a close look at scala-old-collections.